<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.dell.delta_uat_automation.testcases.TC_1341363_LA_Confirm_Service_Details_Override_SLA_Deferrals_UX" tests="1" time="712.190" ignored="0" failures="1" timestamp="28 Mar 2017 07:22:11 GMT" skipped="0" hostname="W1041LQD02" errors="0">
  <testcase name="validate_TC_1341363_LA_Confirm_Service_Details_Override_SLA_Deferrals_UX" time="712.190" classname="com.dell.delta_uat_automation.testcases.TC_1341363_LA_Confirm_Service_Details_Override_SLA_Deferrals_UX">
    <failure type="java.lang.AssertionError" message="Caught Error to Scheduling">
      <![CDATA[java.lang.AssertionError: Caught Error to Scheduling
	at org.testng.Assert.fail(Assert.java:94)
	at com.dell.delta_uat_automation.workflows.NewDispatchWorkFlow.schedule_Day_Dispatch(NewDispatchWorkFlow.java:2760)
	at com.dell.delta_uat_automation.testcases.TC_1341363_LA_Confirm_Service_Details_Override_SLA_Deferrals_UX.validate_TC_1341363_LA_Confirm_Service_Details_Override_SLA_Deferrals_UX(TC_1341363_LA_Confirm_Service_Details_Override_SLA_Deferrals_UX.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at com.dell.delta_uat_automation.utility.TestNGCreator.createXml(TestNGCreator.java:264)
	at com.dell.delta_uat_automation.utility.TestNGCreator.main(TestNGCreator.java:186)
]]>
    </failure>
  </testcase> <!-- validate_TC_1341363_LA_Confirm_Service_Details_Override_SLA_Deferrals_UX -->
</testsuite> <!-- com.dell.delta_uat_automation.testcases.TC_1341363_LA_Confirm_Service_Details_Override_SLA_Deferrals_UX -->
